recipes/bigint/all/conanfile.py

```diff
@@ -1,9 +1,11 @@
-from conan.tools.microsoft import msvc_runtime_flag
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
-import functools
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, get
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.46.0"
```

Comment:
```suggestion
required_conan_version = ">=1.47.0"
```
conan.errors import ConanInvalidConfiguration requires 1.47.0